docs/designers-developers/developers/tutorials/create-block/submitting-to-block-directory.md

```diff
@@ -0,0 +1,102 @@
+# Share your Block with the World
+
+So you've created an awesome block? Care to share?
+
+**Contents**:
+1. Help users understand your block
+2. Analyze your plugin
```

Comment:
What do you think about renaming this section to "Provide data to the block directory"? Something about "Analyze your plugin" makes it sound automated, rather than curating data and describing your block(s)

Reply:
Here are some of my suggestions.

# Share your Block

Share your block with others.

##Contents**:
1. Help users understand your block
2. Analyze your plugin? --> What does that mean? Using basic words to explain what it is.
Does this mean explain what your plugin does? Explain how you made your plugin? 

@ryelle Kelly
Curating data I really do not understand what that means.